lib/profile/profilesRouter.js

```diff
@@ -49,8 +49,8 @@ const cmdDefinitions = [
     handlerName: 'use'
   },
   {
-    command: 'delete <name>',
-    desc: 'Delete profile by name',
+    command: 'delete [name]',
+    desc: 'Delete a specified profile or the active one',
```

Comment:
How about: "... the active one if no profile is specified"

Reply:
I like it better like this. It's not focused so much on the active profile and it just gives you all the options. But it's up to you?

Reply:
OK, leave it like that then.